spdb/project/resource.py

```diff
@@ -34,7 +34,7 @@ def get_isotropic_level(hierarchy_method, x_voxel_size, y_voxel_size, z_voxel_si
             return 0
         else:
             if x_voxel_size != y_voxel_size:
-                raise Exception("X voxel size != Y voxel size. Currently unable to determine isotropic level")
+                raise RuntimeError("X voxel size != Y voxel size. Currently unable to determine isotropic level")
```

Comment:
I think `ValueError` would be more appropriate. https://docs.python.org/3/library/exceptions.html#ValueError

Reply:
I agree, I'll change it